src/megatron/bridge/recipes/__init__.py

```
@@ -21,6 +21,8 @@
 from megatron.bridge.recipes.deepseek import *
 from megatron.bridge.recipes.gemma import *
 from megatron.bridge.recipes.gemma3_vl import *
+from megatron.bridge.recipes.glm import *
```

Flake8 (7.3.0)

[error] 'megatron.bridge.recipes.glm.*' imported but unused

(F401)